inabox/deploy/deploy.go

```diff
@@ -218,7 +218,10 @@ func (env *Config) RunNodePluginBinary(operation string, operator OperatorVars)
 		"NODE_BLS_OPERATOR_STATE_RETRIVER=" + operator.NODE_BLS_OPERATOR_STATE_RETRIVER,
 		"NODE_EIGENDA_SERVICE_MANAGER=" + operator.NODE_EIGENDA_SERVICE_MANAGER,
 		"NODE_CHURNER_URL=" + operator.NODE_CHURNER_URL,
-	})
+		"NODE_NUM_CONFIRMATIONS=0",
```

Comment:
How does this affect the test?

Reply:
Right here : https://github.com/Layr-Labs/eigenda/blob/7f60e15be82c41709f4a072be016b445aba54f95/node/plugin/config.go#L96

Reply:
The anvil will never have 3 confirmations, so then it get stuck in a infinite loop.
https://github.com/Layr-Labs/eigenda/blob/7f60e15be82c41709f4a072be016b445aba54f95/common/geth/client.go#L236

Reply:
I see, thanks